pytorch_tabnet/multitask.py

```diff
@@ -75,7 +75,7 @@ def update_fit_params(self, X_train, y_train, eval_set, weights):
             for classes in self.classes_
         ]
         self.preds_mapper = [
-            {index: class_label for index, class_label in enumerate(classes)}
+            {str(index): str(class_label) for index, class_label in enumerate(classes)}
```

Comment:
This means that we are always treating classes as string?
Is it because it's easier to save?

Reply:
I'm a bit worried that predicting string systematically instead of int constitutes a breaking change.

I'm just trying to understand why this is needed?

Reply:
I do not like it either. This is in order to save as a json file, we either need to handle all possible formats or convert to a single format, in this case strings. I would agree this probably consists a breaking change.  And I do not know what to to in the case the target contains classes `0` (int) and `A` (string), for example.

Reply:
hmmm json files can't handle int and strings? Maybe we should not allow to mix different types. Maybe converting to string is the best option idk.